docs/UserGuide.md

```diff
@@ -33,95 +39,143 @@
 Format: `help`
 
 > Help is also shown if you enter an incorrect command e.g. `abcd`
+
+<br><img src="images/Help.jpeg" width="600"><br>
+
+#### Adding a task in CMDo: `add`
+Adds a task to CMDo <br>
+Format: `add <details> <day> <time> /<priority> -<tag>` 
  
-#### Adding a person: `add`
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` 
- 
-> Words in `UPPER_CASE` are the parameters, items in `SQUARE_BRACKETS` are optional, 
-> items with `...` after them can have multiple instances. Order of parameters are fixed. 
+> You can type anything in details. It must not end with `by`, `on`, `before` or `at`.
 > 
-> Persons can have any number of tags (including 0)
+> For time, typing `tml`, `tmr`, `tomorrow` will still be recognised as tomorrow.
+>
+> For time, typing `1300`, `1pm`, will be recognised as 1300.
+> 
+> For priority, use `/low`, `/medium` or `/high`. Tasks added without specific priority will default to low.
+<br><img src="images/Add1.jpeg" width="600"><br>
+<br><img src="images/Add1a.jpeg" width="600"><br>
+> You can also add a tag to the task by using `-`.
+<br><img src="images/Add2.jpeg" width="600"><br>
+<br><img src="images/Add2a.jpeg" width="600"><br>
+
+#### Listing task in CMDo : `list <day>` or `param` or `/priority`
+Shows a list of all task in the CMDo on that day. It also acts as a filter.<br>
+Format: `list <today>`
+
+> Key in <day> and all task due on that day will appear
+> <br><img src="images/ListTmr.jpeg" width="600"><br>
+> Key in <done> and all done task will appear
+> <br><img src="images/ListDone.jpeg" width="600"><br>
+> Key in any search parameter and list command will add a filter for you
+<br><img src="images/ListTake.jpeg" width="600"><br>
+> Key in </priority> to show all tasks with specified priority
+<br><img src="images/ListHP.jpeg" width="600"><br>
+
+#### Deleting a task in CMDo : `delete`
+Deletes the specified task from the to do list.
+Format: `delete INDEX`
+
+> Deletes the person at the specified `INDEX`. 
+  The index refers to the index number shown in the most recent listing.<br>
+  The index **must be a positive integer** 1, 2, 3, ...
 
 Examples: 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe p/1234567 e/betsycrowe@gmail.com a/Newgate Prison t/criminal t/friend`
+* `list`
+  `delete 2`
+  Deletes the 2nd task in the to do list.
+* `list email` 
+  `delete 1`
+  Deletes the 1st task in the results of the `find` command.
 
-#### Listing all persons : `list`
-Shows a list of all persons in the address book.<br>
-Format: `list`
+#### Update task : `update`
```

Comment:
Since you are using "Edit" would your team consider changing it to Edit instead of Update?